### ISSUE TITLE
errors in primary/secondary color auto-selection

### ISSUE THREAD

**Issue by vecna** [tracking-exposed/hugo-theme-trex#9]
Hi, as far as I understood, based on the website we are running, the hugo system would have given $pimary and $seconday dependently from the website we are running in. But this do not happen:

I include the content as:
`{{<boxes color="secondary" text="The root of evil, a business issue">}}
`
and this is the code embedded:
```
C:\Users\Claudio\VMShared\youtube.tracking.exposed\themes\trex\layouts\shortcodes (master)
λ more boxes.html
<p class="highlight {{ .Get "color"}}">
    {{ .Get "text"}}
</p>
```

as far as I understood, the theme should automatically update the $primary $secondary (and maybe $thirdtiary) color in sass 

but this is not happening, in youtube case, we should have red (and what else?) but is coming back the yellow and the greenish: https://testing.tracking.exposed/seeyou/

**Comment by lc-d** [comment]
Got it!

**Comment by lc-d** [comment]
fixed on b13cbec

**Comment by privatemaker** [comment]
FWIW, forcing things into using `$primary` and `$secondary` color is not the way I architected the SASS / CSS. to make things look good across sites, this is not enough. hence why I used nested styles in `_components.scss` and such
